WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardService.swift

```diff
@@ -85,4 +87,18 @@ private extension BlogDashboardService {
         decoder.keyDecodingStrategy = .convertFromSnakeCase
         return try? decoder.decode(BlogDashboardRemoteEntity.self, from: data)
     }
+
+    func localCardsAndGhostCards() -> DashboardSnapshot {
+        var snapshot = parse([:], cards: BlogDashboardRemoteEntity())
+
+        let section = DashboardCardSection(id: .ghost)
+
+        snapshot.appendSections([section])
+        Array(0...4).forEach {
+            snapshot.appendItems([DashboardCardModel(id: .ghost,
+                                                     apiResponseDictionary: ["diff": $0])], toSection: section)
```

Comment:
[Q] Are you passing ["diff": $0] in the apiResponseDictionary to make diffable data source recognize these cards as unique cards?

Reply:
Yes, and this is a good point. I'll refactor the name of this property, this should basically be a dictionary that we use to differentiate cards. It can be an API response or anything else.

As is named right now it makes this part read really strange.

Reply:
Done in 5d6807c874fab07b22eab08f7435355dcbfb7c52